params/config.go

```diff
@@ -173,82 +174,68 @@ var (
-			TokenMint:               (*math.HexOrDecimal256)(common.Big0),
+			WithholdingThreshold:     0,   // 0%, no tolerance
+			ProposerRewardRate:       500, // 5%
+			OracleRewardRate:         500, // 5%
+			SkipGenesisVerification:  false,
+			// Schedules are set in the DefaultBakerlooGenesisBlock
+			MaxScheduleDuration: uint64(4*SecondsInYear + SecondsInDay),
+			ClusteringThreshold: 100,
+			Validators:          BakerlooValidators,
+			TokenBond:           (*math.HexOrDecimal256)(new(big.Int).Mul(big.NewInt(1_620_027), NtnPrecision)),
+			TokenMint:           (*math.HexOrDecimal256)(new(big.Int).Mul(big.NewInt(60_000_000), NtnPrecision)),
 		},
 		OracleContractConfig: &OracleContractGenesis{
-			VotePeriod: OracleVotePeriod,
-			Symbols:    OracleInitialSymbols,
+			Symbols:                   []string{"AUD-USD", "CAD-USD", "EUR-USD", "GBP-USD", "JPY-USD", "SEK-USD"},
+			VotePeriod:                600,
+			OutlierDetectionThreshold: 3,
+			OutlierSlashingThreshold:  225,
+			BaseSlashingRate:          10,
+			NonRevealThreshold:        5,
+			RevealResetInterval:       10,
+			SlashingRateCap:           50,
 		},
 		ASM: AsmConfig{
 			ACUContractConfig:           DefaultAcuContractGenesis,
 			StabilizationContractConfig: DefaultStabilizationGenesis,
 			SupplyControlConfig: &SupplyControlGenesis{
-				InitialAllocation: (*math.HexOrDecimal256)(new(big.Int).Sub(
-					new(big.Int).Exp(big.NewInt(2), big.NewInt(256), nil), // 2^256
-					new(big.Int).Mul(big.NewInt(16), big.NewInt(Ether)),   // 16 * 10^18
-				)),
+				InitialAllocation: (*math.HexOrDecimal256)(
+					new(big.Int).Sub(
+						new(big.Int).Sub(
+							new(big.Int).Exp(big.NewInt(2), big.NewInt(256), nil), // 2^256
+							new(big.Int).Mul(big.NewInt(403), big.NewInt(Ether)),
+						),
+						common.Big1,
+					),
+				),
 			},
+			AuctioneerContractConfig: DefaultAuctioneerGenesis,
 		},
 		AccountabilityConfig:         DefaultAccountabilityConfig,
 		OmissionAccountabilityConfig: DefaultOmissionAccountabilityConfig,
+		InflationContractConfig: &InflationControllerGenesis{
+			InflationRateInitial:      (*math.HexOrDecimal256)(new(big.Int).Div(new(big.Int).Mul(big.NewInt(75), DecimalFactor), big.NewInt(1000*SecondsInYear))),        // 7.5% AR
+			InflationRateTransition:   (*math.HexOrDecimal256)(new(big.Int).Div(new(big.Int).Mul(big.NewInt(55), DecimalFactor), big.NewInt(1000*SecondsInYear))),        // 5.5% AR
+			InflationReserveDecayRate: (*math.HexOrDecimal256)(new(big.Int).Div(new(big.Int).Mul(big.NewInt(17_317), DecimalFactor), big.NewInt(100_000*SecondsInYear))), // 17.3168186793% AR
+			InflationTransitionPeriod: (*math.HexOrDecimal256)(new(big.Int).Mul(big.NewInt(4*SecondsInYear+SecondsInDay), DecimalFactor)),                                // (4+1/365) years
+			InflationCurveConvexity:   (*math.HexOrDecimal256)(new(big.Int).Div(new(big.Int).Mul(big.NewInt(2_765), DecimalFactor), big.NewInt(1_000))),                  // -1.7794797758
```

Comment:
```suggestion
			InflationCurveConvexity:   (*math.HexOrDecimal256)(new(big.Int).Div(new(big.Int).Mul(big.NewInt(2_765), DecimalFactor), big.NewInt(1_000))),                  // 2.7648007374
```

Commented value out of date